api/src/contexts/users/index.ts

```diff
@@ -23,25 +22,25 @@ const prisma = new PrismaClient();
 const userRepository = new UserRepository(prisma);
 
 const createUserInteractor = new CreateUserInteractor(userRepository);
-// const updateUserInteractor = new UpdateUserProfileInteractor(userRepository);
+const updateUserInteractor = new UpdateUserProfileInteractor(userRepository);
 const deleteUserInteractor = new DeleteUserInteractor(userRepository);
 
 // Controllers
 
 const createUserController = new CreateUserController(createUserInteractor);
-// const updateUserProfileController = new UpdateUserProfileController(
-//   updateUserInteractor,
-// );
+const updateUserProfileController = new UpdateUserProfileController(
+  updateUserInteractor,
+);
 const deleteUserController = new DeleteUserController(deleteUserInteractor);
 
 // Routes
 UsersRouter.post(
   '/signup',
   createUserController.execute.bind(createUserController),
 );
-// UsersRouter.put('/users/:id', verifyAccessToken, (req, res) => {
-//   updateUserProfileController.execute(req, res);
-// });
+UsersRouter.put('/users/:id', verifyAccessToken, (req, res) => {
```

Comment:
自分以外のユーザーのidを指定してプロフィール更新できちゃうと問題ありそうなので、パラメータとしてidを受け付けるのをやめたいです！
```suggestion
UsersRouter.put('/profile', verifyAccessToken, (req, res) => {
```